cypress/fixtures/capabilities.json

```diff
@@ -0,0 +1 @@
+{"provider_type":"remote","package_url":"https://github.com/layer5labs/meshery-extensions-packages/releases/download/v0.6.5.4.3/provider-meshery.tar.gz","provider_name":"Meshery","provider_description":["Persistent sessions","Save environment configurations and user preferences","Store and retrieve patterns, filters, and application defintions","Retrieve performance and conformance test results","RBAC support","Free use"],"provider_url":"https://meshery.layer5.io","extensions":{"navigator":[{"title":"MeshMap","on_click_callback":1,"href":{"uri":"/meshmap","external":false},"component":"/provider/navigator/meshmap/index.js","icon":"/provider/navigator/img/meshmap-icon.svg","link":true,"show":true,"type":"full_page"}],"user_prefs":[{"component":"/provider/userpref/meshmap_userpref/index.js","type":"component"}],"graphql":[{"component":"meshmap","path":"provider/navigator/meshmap/graphql/plugin.so","type":"backend"}],"account":[{"title":"Account","on_click_callback":1,"href":{"uri":"/account","external":false},"component":"/provider/account/profile/index.js","link":true,"show":true,"type":"full_page"}]},"capabilities":[{"feature":"sync-prefs","endpoint":"/user/preferences"},{"feature":"persist-results","endpoint":"/results"},{"feature":"persist-result","endpoint":"/result"},{"feature":"persist-smi-results","endpoint":"/smi/results"},{"feature":"persist-smi-result","endpoint":"/smi/result"},{"feature":"persist-metrics","endpoint":"/result/metrics"},{"feature":"persist-smp-test-profile","endpoint":"/user/test-config"},{"feature":"persist-performance-profiles","endpoint":"/user/performance/profiles"},{"feature":"persist-schedules","endpoint":"/user/schedules"},{"feature":"persist-meshery-patterns","endpoint":"/patterns"},{"feature":"persist-meshery-filters","endpoint":"/filters"},{"feature":"persist-meshery-applications","endpoint":"/applications"},{"feature":"persist-meshery-pattern-resources","endpoint":"/patterns/resource"}],"package_version":"v0.6.5.4.3"}
```

Comment:
I wonder if this fake version will cause an accidental issue.

Reply:
No but it can definitely create confusions in future. The version is overrides dynamically based on the latest release, but making the change so not to confuse our future self.

Reply:
Ok, gotcha.